modules/tests/src/test/scala/higherkindness/mu/rpc/protocol/RPCTests.scala

```diff
@@ -78,105 +75,6 @@ class RPCTests extends RpcBaseTestSuite with BeforeAndAfterAll {
-      clientProgram[ConcurrentMonad]("SRE")
-        .unsafeRunSync() shouldBe C("INVALID_ARGUMENT: SRE", a1)
-      clientProgram[ConcurrentMonad]("RTE")
-        .unsafeRunSync() shouldBe C("INTERNAL: RTE", a1)
-      clientProgram[ConcurrentMonad]("Thrown")
-        .unsafeRunSync() shouldBe C("UNKNOWN", a1)
-    }
-
-    "be able to run unary services with avro schema" in {
-
-      def clientProgram[F[_]](implicit APP: MyRPCClient[F]): F[C] =
-        APP.uws(a1.x, a1.y)
-
-      clientProgram[ConcurrentMonad].unsafeRunSync() shouldBe c1
-
-    }
-
-    "be able to run server streaming services" in {
-
-      def clientProgram[F[_]](implicit APP: MyRPCClient[F]): F[List[C]] =
-        APP.ss(a2.x, a2.y)
-
-      clientProgram[ConcurrentMonad].unsafeRunSync() shouldBe cList
-
-    }
-
-    "handle errors in server streaming services" in {
-
-      def clientProgram[F[_]](
-          errorCode: String
-      )(implicit APP: MyRPCClient[F], M: MonadError[F, Throwable]): F[List[C]] =
-        M.handleError(APP.sswe(a1, errorCode))(ex => List(C(ex.getMessage, a1)))
-
-      clientProgram[ConcurrentMonad]("SE")
-        .unsafeRunSync() shouldBe List(C("INVALID_ARGUMENT: SE", a1))
-      clientProgram[ConcurrentMonad]("SRE")
-        .unsafeRunSync() shouldBe List(C("INVALID_ARGUMENT: SRE", a1))
-      clientProgram[ConcurrentMonad]("RTE")
-        .unsafeRunSync() shouldBe List(
-        C("UNKNOWN", a1)
-      ) //todo: consider preserving the exception as is done for unary
-      clientProgram[ConcurrentMonad]("Thrown")
-        .unsafeRunSync() shouldBe List(C("UNKNOWN", a1))
-    }
-
-    "be able to run client streaming services" in {
-
-      def clientProgram[F[_]](implicit APP: MyRPCClient[F]): F[D] =
-        APP.cs(cList, i)
-
-      clientProgram[ConcurrentMonad].unsafeRunSync() shouldBe dResult
-    }
-
-    "be able to run client bidirectional streaming services" in {
-
-      ignoreOnTravis("TODO: restore once https://github.com/higherkindness/mu/issues/281 is fixed")
-
-      def clientProgram[F[_]](implicit APP: MyRPCClient[F]): F[E] =
-        APP.bs(eList)
-
-      clientProgram[ConcurrentMonad].unsafeRunSync() shouldBe e1
-
-    }
-
-    "be able to run client bidirectional streaming services with avro schema" in {
-
-      def clientProgram[F[_]](implicit APP: MyRPCClient[F]): F[E] =
-        APP.bsws(eList)
-
-      clientProgram[ConcurrentMonad].unsafeRunSync() shouldBe e1
-
-    }
-
-    "be able to run rpc services monadically" in {
-
-      def clientProgram[F[_]: Monad](implicit APP: MyRPCClient[F]): F[(C, C, List[C], D, E, E)] = {
-        for {
-          u <- APP.u(a1.x, a1.y)
-          v <- APP.uws(a1.x, a1.y)
-          w <- APP.ss(a2.x, a2.y)
-          x <- APP.cs(cList, i)
-          y <- APP.bs(eList)
-          z <- APP.bsws(eList)
-        } yield (u, v, w, x, y, z)
-      }
-
-      clientProgram[ConcurrentMonad].unsafeRunSync() shouldBe ((c1, c1, cList, dResult, e1, e1))
-
-    }
```

Comment:
❓ Aren't these tests useful anymore?

Reply:
They're testing methods that I deleted which were running monix code. I need to see if the methods should be put back and rewritten using fs2